src/components/Navbar.js

```diff
@@ -1,24 +1,24 @@
-import React from 'react'; 
-import {NavLink} from 'react-router-dom'; 
+import React from 'react';
+import { NavLink } from 'react-router-dom';
 
-function Navbar(){
+function Navbar() {
     return(
         <nav className="navbar navbar-expand-lg navbar-light bg-dark">
-      
           <ul className="navbar-nav m-auto">
             <li className="nav-item">
                 <NavLink className="nav-link text-white text-uppercase ml-5" exact to="/data" activeClassName="active">
-                    Visualization&nbsp; 
```

Comment:
I would also try to avoid using non-breaking spaces; if you need extra space, css padding can help